aws-redshiftserverless-workgroup/src/main/java/software/amazon/redshiftserverless/workgroup/UpdateHandler.java

```diff
@@ -102,6 +102,7 @@ private ResourceModel getUpdatableResourceModel(ResourceModel desiredModel, Reso
 
         return desiredModel.toBuilder()
                 .baseCapacity((Integer) getDelta.apply(desiredModel.getBaseCapacity(), previousModel.getBaseCapacity()))
+                .maxCapacity((Integer) getDelta.apply(desiredModel.getMaxCapacity(), previousModel.getMaxCapacity()))
```

Comment:
what does this line do, is it possible getDelta returns a negative value, and can we handle it?

Reply:
In the getDelta function, it has
```
return Objects.equals(desired, previous) ? null : desired;
```
If the desired model different from the current model, it will use the new model data